src/dev-app/google-map/google-map-demo.ts

```diff
@@ -40,4 +46,12 @@ export class GoogleMapDemo {
   handleRightclick() {
     this.markerPositions.pop();
   }
+
+  displayPolyline() {
+    this.isPolylineDisplayed = !this.isPolylineDisplayed;
+  }
+
+  editablePolyline() {
+    this.polylineOptions = {...this.polylineOptions, editable: true};
```

Comment:
Should this be a toggle as well? Or should we disable the button after the first time it's clicked?

Reply:
Maybe this should be a checkbox as well.

Reply:
Done